### ISSUE TITLE
Why version 0.X?

### ISSUE THREAD

**Issue by julealgon** [force-net/DeepCloner#23]
This library has a significant amount of downloads and appears to be stable and reliable. Why does it still have major version 0?

This makes the library look rather unprofessional to consumers and actually raised a red flag when I saw it being added in one of our projects. Usually version 0.X are reserved for alpha and beta versions, basically things that are not yet ready to market.

Can the next release bump the version to 1.0.0 if the library is considered stable and production-ready?

**Comment by force-net** [comment]
Approach with 0.x version is acceptable (there are some pros for this). But yes, I need to switch it to 1.0.0
I'll try to implement one feature related to dictionary cloning, it can break the API. Or... if I do not find the time to do that, I'll switch last released version version to 1.0.0

**Comment by SvenSchlesinger** [comment]
Are there any updates on this one?  As @julealgon mentioned, the library is stable and realiable. Unfortunately, as long as it is in version 0.x, I cannot use the library in one of my projects and I would have to switch to another one.

**Comment by niemyjski** [comment]
I agree it should be bumped, but it's crazy you won't use it because it's not 1.0...
